mods/ts/installer/firstdecade.yaml

```
@@ -10,8 +10,7 @@ tfd: C&C The First Decade (English)
 			Extract:
 				^SupportDir|Content/ts/scores.mix: Tiberian Sun\SUN\SCORES.MIX
 				^SupportDir|Content/ts/tibsun.mix: Tiberian Sun\SUN\TIBSUN.MIX
-				^SupportDir|Content/ts/expand01.mix: Tiberian Sun\SUN\expand01.mix
-				^SupportDir|Content/ts/scores01.mix: Tiberian Sun\SUN\scores01.mix
+				^SupportDir|Content/ts/firestorm/scores01.mix: Tiberian Sun\SUN\scores01.mix
```

You removed `expand01.mix` instead of changing its destination.
```suggestion
				^SupportDir|Content/ts/firestorm/expand01.mix: Tiberian Sun\SUN\expand01.mix
				^SupportDir|Content/ts/firestorm/scores01.mix: Tiberian Sun\SUN\scores01.mix
```

Don't forget to change the destination also on the `extract-raw: ^SupportDir|Content/ts/firestorm/expand01.mix` and `delete: ^SupportDir|Content/ts/firestorm/expand01.mix` statements!

---

No need to change the temporary `expand01.mix` location - this is just being used as a temporary working directory because we can't extract direct from the cab, so `ts` is fine.